packages/tradershub/src/features/cfd/screens/Jurisdiction/JurisdictionCard/JurisdictionCard.tsx

```diff
@@ -91,13 +91,14 @@ const JurisdictionCard = ({ isAdded = false, isSelected = false, jurisdiction, o
             return row.clickableDescription.map(description => {
                 if (description.type === 'link') {
                     return (
-                        <Link
-                            className='py-0 pl-0 text-sm text-brand-red-light hover:no-underline'
+                        <Button
```

Comment:
@thisyahlen-deriv Why this changed to button?
it changes the expected behaviour on hover.

Reply:
Because Text has no onclick. And there is onclick, i cant use div or a tag cause sonarcloud will complain

Reply:
can use static link